src/test/java/org/prebid/server/bidder/telaria/TelariaBidderTest.java

```diff
@@ -135,7 +135,7 @@ public void makeHttpRequestsShouldNotSetAppPublisherIdIfSiteIsNotNull() {
         // given
         final BidRequest bidRequest = BidRequest.builder()
                 .imp(singletonList(givenImp(identity())))
-                .site(Site.builder().build())
+                .site(Site.builder().publisher(Publisher.builder().build()).build())
```

Comment:
Is it really good idea for adding empty publisher and hiding NPE in the test ?  

Fix NPE remove this